Makefile

```diff
@@ -304,7 +304,7 @@ cns-image-windows:
 	$(MKDIR) $(IMAGE_DIR); 
 	docker build \
 	--no-cache \
-	-f cns/windows.Dockerfile \
+	-f cns/Dockerfile.windows \
```

Comment:
I prefer it the other way around, `Dockerfile` is the type of file and therefore goes as the extension

Reply:
ok I'll renamed the npm one